package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ember-cli-deploy-redis",
   "version": "0.0.0",
-  "description": "The default blueprint for ember-cli addons.",
+  "description": "Ember CLI Deploy plugin to deploy index.html into redis.",
```

Comment:
@lukemelia I'm keen to not be specific about index.html as this plugin could be used to upload anything to redis.  Thoughts?